server.js

```diff
@@ -254,7 +254,9 @@ app.prepare().then(() => {
     server.use(expressSitemapXml(getSitemapURLs(apiServer), 'https://app.co'))
 
     server.get('/robots.txt', async (req, res) => {
-      const robots = await fs.readFile('./static/robots.txt')
+      const robotsFile = dev ? 'robots.staging.txt': 'robots.txt'
```

Comment:
This is good, but our staging environments have `NODE_ENV=production`, so `dev` will be `false` here. Instead, we also have an ENV variable `STAGING` on our staging servers. So, if you also add a check for `process.env.STAGING`, this will be perfect.